apps/i18n/common/en_us.json

```diff
@@ -2633,6 +2633,7 @@
   "startOverTitle": "Are you sure you want to start over?",
   "startOverBody": "This will reset the puzzle to its start state and reset all the data you've added or changed.",
   "startOverWorkspace": "This will reset the workspace to its start state and remove all the blocks you've added or changed.",
+  "startOverWorkspaceText": "This will reset the workspace to its start state and remove all the code you've added or changed.",
```

Comment:
This is maybe a product question, but I wonder if we could just use this string (referencing "code") for all labs, since the blocks are the code.

Reply:
I kept them separate because Music Lab uses this string and they are in a string lock now.

Reply:
Good point!